src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__img {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
```

Comment:
Avoid using tag selectors for styling. It's better to style using classes, IDs, pseudo-classes, pseudo-elements, and attributes. This is a code quality issue. Tag selectors have a high specificity and can lead to problems with overriding styles later on. Plus, they can accidentally style unwanted elements. In this case, you should add a class to the `ul` element and style that class instead.

Reply:
<ul class="nav__list">